vm-virtio/src/vhost_user/mod.rs

```diff
@@ -11,12 +11,17 @@ use std;
 use std::io;
 use vhost_rs::Error as VhostError;
 use vm_memory::Error as MmapError;
+use vm_memory::GuestMemoryError;
 
 mod handler;
 pub mod net;
 
+pub use self::net::CtlVirtqueue;
 pub use self::net::Net;
 
+pub const VIRTIO_NET_CTRL_MQ_VQ_PAIRS_MIN: u16 = 1;
+pub const VIRTIO_NET_CTRL_MQ_VQ_PAIRS_MAX: u16 = 0x8000;
+
```

Comment:
MSI-X Table entry number, from 0 to 0x7FF, we can make those definition more general.